app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -21,6 +21,11 @@ class RawFileManager extends RawFileBrowser {
     this.handleActionBarAddFolderClick(event);
   };
 
+  onActionBarSubmitURLClick = (event, selectedItem) => {
+    event.preventDefault();
+    this.props.onActionBarSubmitURLClick(selectedItem);
```

Comment:
I don't think we need to pass the selected item do we? It doesn't/shouldn't matter if an item is selected when we upload a URL.